src/Ocelot/Configuration/File/FileMetadataOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Globalization;
+
+namespace Ocelot.Configuration.File;
+
+public class FileMetadataOptions
```

Comment:
Name can be shorter because of `File` prefix.
Also there are no XML docs of the feature the class is a part of.